DTML.EduBot/Dialogs/LessonDialog.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using AdaptiveCards;
+using DTML.EduBot.LessonPlan;
+using Microsoft.Bot.Connector;
+
+namespace DTML.EduBot.Dialogs
+{
+    using System.Threading.Tasks;
```

Comment:
can you either move the using statements inside or outside the namespace? currently they are mixed.